autoload/argwrap.vim

```diff
@@ -187,8 +187,19 @@ function! argwrap#wrapContainer(range, container, arguments, wrapBrace, tailComm
         end
 
         call append(l:line, l:text)
-        let l:line += 1
-        silent! exec printf('%s>', l:line)
+
+        if !a:wrapBrace
```

Comment:
```suggestion
        if !a:wrapBrace && !&expandtab
```

Disable the feature if the `expandtab` is set.
Otherwise you would have to decide which are the rules: align all parameters to the next tab position ? Should the first one be aligned too ?

Reply:
Hm, I don't know exactly what you mean, but when I add the `&& !expandtab`, this behaviour occurs on `func(1, 2. 3)`:

```python
# with expandtab off (the behaviour we wish)
func(1,
     2,
     3)

# with expandtab on (not someting we want)
func(
    1,
    2,
    3)
```

I think in both cases (expandtab on or off), the first argument wrap is desired, right? Or do you think other behaviour is desired when expandtab is on?

Reply:
What I meant is since your feature only work with spaces, when using tabs it should be disabled.
So that the previous behavior is preserved.
But I never used this kind of alignment because I think it's defeat the point.
IMO the main reasons to wrap are:
- To avoid long lines: if the function name is long and the parameters are long too, string for examples, then it defeat the purpose
- To avoid messy diffs (modifying one line should only print one line in the diff): with this kind of alignment to add a new parameter you must first modify the previous one

Reply:
Sure, I get your points. I'll close this pull-request for now. Thank you for the clarity!

Reply:
For python,  I never use tab, only use space. 
I like
```
void foo(var1,
         var2,
         var3)
```

Any problem with this setting? Why do you close it?